nextflow_schema.json

```diff
@@ -206,12 +207,14 @@
                     "type": "boolean",
                     "description": "Display help text.",
                     "fa_icon": "fas fa-question-circle",
+                    "default": false,
```

Comment:
You can remove all `"default": false`, this is taken into account by default and will be removed if you use `nf-core schema build`

Reply:
It didn't automatically remove the defaults, but I removed them manually now.